### PR TITLE
fix(duckduckgo): unthemed elements

### PR DESCRIPTION
**Pull request by thismoon**
## 🔧 What does this fix? 🔧

- new accent color variable
image
- tooltip background color
image
- menu item hover background
image
- code background
[example page](https://duckduckgo.com/?q=python+how+to+split+a+string)
search page:
before:
image
after:
image

ai chat:
before:
image
after:
image

- this border
image
- unexpanded lyrics gradient and explicit tag
[example page](https://duckduckgo.com/?q=pride+kendrick+lyrics)
image

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/duckduckgo/catppuccin.user.css [hunk 1]
```diff
@@ -448,6 +451,14 @@
     .module__inner-toggle::after {
       background-color: @surface2 !important;
     }
+    .module--lyrics:not(.is-expanded)
+      .module--lyrics__footer.can-expand::after {
+      background: linear-gradient(rgba(0, 0, 0, 0), @surface0);
```
Comment (uncenter) [inline styles/duckduckgo/catppuccin.user.css:456]: Can we avoid using a non-palette color in this gradient?

Comment (thismoon) [reply styles/duckduckgo/catppuccin.user.css:456]: i used it after checking that there are other non palette colors in the style
image
there are 6 other occurrences of using an rgba for an invisible color
it's an invisible color with 0 opacity so there's no difference between it and `fade(@surface0, 0%)`. also when reading the code `rgba(0, 0, 0, 0)` is clearer and feels more like an invisible color while the other form feels more of a palette color that has an opacity
but that's my opinion, if you really think that using fade with a palette color and zero percentage is more appropriate i'll change it

Comment (uncenter) [reply styles/duckduckgo/catppuccin.user.css:456]: Ah I didn't see it was 0. Why not just use `transparent` for clarity?

Comment (thismoon) [reply styles/duckduckgo/catppuccin.user.css]: oh i forgot that i can do that. i replaced all the transparent colors with `transparent`. is it alright now?

**Review by GenShibe — APPROVED** [review]

**Pull request merged**